app/src/main/java/org/breezyweather/common/snackbar/Snackbar.kt

```diff
@@ -332,19 +331,12 @@ class Snackbar private constructor(
                 attrs,
                 com.google.android.material.R.styleable.SnackbarLayout
             )
-            mMaxWidth = a.getDimensionPixelSize(
-                com.google.android.material.R.styleable.SnackbarLayout_android_maxWidth,
-                -1
-            )
```

Comment:
`com.google.android.material.R.styleable.SnackbarLayout_android_maxWidth` is never set. As a result, `mMaxWidth` is always -1 and never used.

Reply:
I think at some point we will need to move to a more native implementation of the Snackbar